packages/mail/lib/getNewsletterSettings.js

```diff
@@ -31,13 +34,16 @@ module.exports = async ({
 
   const status = member.status
   const subscriptions = []
-  supportedInterestIds.forEach(interestId => {
-    // only return already configured interests / subscriptions
-    if (interestId in member.interests) {
+  supportedInterestConfigs.forEach(({ interestId, visibleToRoles }) => {
+    // only return visible interests
+    const subscribed = status === MailchimpInterface.MemberStatus.Subscribed
+      ? (!!member.interests[interestId] || false)
```

Comment:
`!!` seems to make `|| false` obsolete here:

```suggestion
      ? !!member.interests[interestId]
```